src/resources/js/tribe-events-ajax-calendar.js

```diff
@@ -402,12 +402,13 @@
 					eventDate: ts.date
 				};
 
+				ts.url_params = {};
+
 				if ( ts.category ) {
 					ts.params['tribe_event_category'] = ts.category;
```

Comment:
since we are inside of js: `ts.params['tribe_event_category']` to `ts.params.tribe_event_category`